public/electron.js

```diff
@@ -81,10 +82,24 @@ function disableDetachedMode() {
   mainWindow && mainWindow.setIgnoreMouseEvents(false);
 }
 
+function checkAndDownloadUpdate() {
+  autoUpdater.setFeedURL({
+    provider: 'github',
+    owner: 'kamranahmedse',
+    protocol: 'https',
+    repo: 'pennywise',
+  });
```

Comment:
This is not required. [According to the docs](https://www.electron.build/auto-update#quick-setup-guide):

> Do not call setFeedURL. `electron-builder` automatically creates `app-update.yml` file for you on build in the resources (this file is internal, you don’t need to be aware of it).

Reply:
OK. Thank you. I missed that. 😢 

I delete unnecessary code.